files/ko/web/http/headers/referer/index.html

```diff
@@ -44,12 +47,16 @@ <h2 id="디렉티브">디렉티브</h2>
 
 <dl>
  <dt>&lt;url&gt;</dt>
- <dd>현재 요청된 페이지의 링크 이전의 웹 페이지의 절대 혹은 부분 주소. URL 프래그먼트(예를 들어, "#section")는 포함되지 않습니다.</dd>
+ <dd>현재 요청된 페이지의 링크 이전의 웹 페이지의 절대 혹은 부분 주소. URL 프래그먼트(예를 들어, "#section")는 포함되지 않습니다.</dd>
```

Comment:
원문과 비교하여 뒷부분이 생략된 것 같아 추가 부탁드립니다!

Reply:
수정부분 모두 적용시켰습니다. 피드백감사합니다!

Reply:
영어 원문에서 뒤에 좀 더 추가적으로 설명하는 부분이 있어 이것도 포함하면 좋을 거 같아 제안 드립니다 :)

```suggestion
 <dd>현재 요청된 페이지의 링크 이전의 웹 페이지의 절대 혹은 부분 주소. URL 프래그먼트(예를 들어, "#section")나 사용자 정보(예를 들어 "https://username:password@example.com/foo/bar/" 에서 "username:password")는 포함되지 않습니다. 오리진, 패쓰, 쿼리스트링은 <a href="/ko/docs/Web/HTTP/Headers/Referrer-Policy#directives">referrer 정책</a>에 따라 포함될 수 있습니다.</dd>
```